src/solvers/contract.jl

```diff
@@ -0,0 +1,19 @@
+function contract_updater(
+  init;
+  state!,
+  projected_operator!,
+  outputlevel,
+  which_sweep,
+  sweep_plan,
+  which_region_update,
+  region_kwargs,
+  updater_kwargs,
+)
+  v = ITensor(1.0)
```

Comment:
```suggestion
  v = ITensor(true)
```
to be more agnostic about the element type (`1.0` is `Float64` which would then convert single precision calculations to double precision, for example).